prow/cmd/deck/main.go

```diff
@@ -1150,7 +1150,36 @@ func handleProwJob(prowJobClient prowv1.ProwJobInterface) http.HandlerFunc {
 	}
 }
 
-func handleRerun(prowJobClient prowv1.ProwJobInterface, createProwJob bool) http.HandlerFunc {
+// canTriggerJob determines whether the given user can trigger any job.
+func canTriggerJob(user string, cfg config.RerunAuthConfig) bool {
+	if cfg.AllowAnyone {
+		return true
+	}
+	if cfg.AuthorizedUsers == nil {
+		return false
+	}
+	for _, s := range cfg.AuthorizedUsers {
+		if user == s {
```

Comment:
Please use `github.NormLogin` to normalize `user` (before the loop) and also for each `s`.